page 20220
/bbs/Gossiping/M.1495067796.A.E2B.html download to ptt/Gossiping/20220_0.txt, time: 6.250000
/bbs/Gossiping/M.1495067837.A.D60.html download to ptt/Gossiping/20220_1.txt, time: 5.060000
/bbs/Gossiping/M.1495067863.A.2BD.html download to ptt/Gossiping/20220_2.txt, time: 4.300000
/bbs/Gossiping/M.1495067912.A.D65.html download to ptt/Gossiping/20220_3.txt, time: 3.170000
/bbs/Gossiping/M.1495067914.A.1A4.html download to ptt/Gossiping/20220_4.txt, time: 4.790000
/bbs/Gossiping/M.1495068026.A.0D5.html download to ptt/Gossiping/20220_5.txt, time: 5.180000
/bbs/Gossiping/M.1495068082.A.808.html download to ptt/Gossiping/20220_6.txt, time: 8.560000
/bbs/Gossiping/M.1495068175.A.340.html download to ptt/Gossiping/20220_7.txt, time: 4.600000
/bbs/Gossiping/M.1495068209.A.024.html download to ptt/Gossiping/20220_8.txt, time: 5.110000
/bbs/Gossiping/M.1495068309.A.2C6.html download to ptt/Gossiping/20220_9.txt, time: 6.100000
/bbs/Gossiping/M.1495068406.A.A03.html download to ptt/Gossiping/20220_10.txt, time: 6.640000
/bbs/Gossiping/M.1495068460.A.7EB.html download to ptt/Gossiping/20220_11.txt, time: 5.850000
/bbs/Gossiping/M.1495068482.A.FAD.html download to ptt/Gossiping/20220_12.txt, time: 4.340000
/bbs/Gossiping/M.1495068512.A.4FD.html download to ptt/Gossiping/20220_13.txt, time: 4.390000
/bbs/Gossiping/M.1495068513.A.AE7.html download to ptt/Gossiping/20220_14.txt, time: 5.750000
/bbs/Gossiping/M.1495068524.A.3C3.html download to ptt/Gossiping/20220_15.txt, time: 4.940000
/bbs/Gossiping/M.1495068527.A.523.html download to ptt/Gossiping/20220_16.txt, time: 5.080000
/bbs/Gossiping/M.1495068559.A.18C.html download to ptt/Gossiping/20220_17.txt, time: 6.210000
/bbs/Gossiping/M.1495068576.A.8DE.html download to ptt/Gossiping/20220_18.txt, time: 3.580000
/bbs/Gossiping/M.1495068586.A.45B.html download to ptt/Gossiping/20220_19.txt, time: 3.890000
page 20219
/bbs/Gossiping/M.1495066711.A.1E1.html download to ptt/Gossiping/20219_0.txt, time: 6.080000
/bbs/Gossiping/M.1495066728.A.96D.html download to ptt/Gossiping/20219_1.txt, time: 3.330000
/bbs/Gossiping/M.1495066797.A.322.html download to ptt/Gossiping/20219_2.txt, time: 4.210000
/bbs/Gossiping/M.1495066849.A.145.html download to ptt/Gossiping/20219_3.txt, time: 3.870000
/bbs/Gossiping/M.1495066867.A.04D.html download to ptt/Gossiping/20219_4.txt, time: 4.580000
/bbs/Gossiping/M.1495066883.A.438.html download to ptt/Gossiping/20219_5.txt, time: 4.410000
/bbs/Gossiping/M.1495066975.A.EF6.html download to ptt/Gossiping/20219_6.txt, time: 5.960000
/bbs/Gossiping/M.1495067002.A.9E5.html download to ptt/Gossiping/20219_7.txt, time: 5.840000
/bbs/Gossiping/M.1495067070.A.8DE.html download to ptt/Gossiping/20219_8.txt, time: 3.870000
/bbs/Gossiping/M.1495067116.A.9FD.html download to ptt/Gossiping/20219_9.txt, time: 5.840000
/bbs/Gossiping/M.1495067124.A.5F4.html download to ptt/Gossiping/20219_10.txt, time: 3.590000
/bbs/Gossiping/M.1495067243.A.4A4.html download to ptt/Gossiping/20219_11.txt, time: 6.860000
/bbs/Gossiping/M.1495067303.A.A81.html download to ptt/Gossiping/20219_12.txt, time: 4.700000
/bbs/Gossiping/M.1495067381.A.19C.html download to ptt/Gossiping/20219_13.txt, time: 7.010000
/bbs/Gossiping/M.1495067510.A.EA8.html download to ptt/Gossiping/20219_14.txt, time: 4.930000
/bbs/Gossiping/M.1495067580.A.388.html download to ptt/Gossiping/20219_15.txt, time: 4.310000
/bbs/Gossiping/M.1495067580.A.F23.html download to ptt/Gossiping/20219_16.txt, time: 5.460000
/bbs/Gossiping/M.1495067588.A.B15.html download to ptt/Gossiping/20219_17.txt, time: 4.950000
/bbs/Gossiping/M.1495067647.A.FEF.html download to ptt/Gossiping/20219_18.txt, time: 5.870000
/bbs/Gossiping/M.1495067757.A.6E1.html download to ptt/Gossiping/20219_19.txt, time: 3.700000
page 20218
/bbs/Gossiping/M.1495065520.A.D9F.htmlTraceback (most recent call last):
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 386, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 382, in _make_request
    httplib_response = conn.getresponse()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/adapters.py", line 438, in send
    timeout=timeout
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 649, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 386, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 382, in _make_request
    httplib_response = conn.getresponse()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
requests.packages.urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./crawler.py", line 177, in <module>
    main(sys.argv[1:])
  File "./crawler.py", line 158, in main
    downloadForum(forumName, startPage, pages)
  File "./crawler.py", line 140, in downloadForum
    downloadPage(str(articleCount), forumName, pageNum, 'https://www.ptt.cc'+link)
  File "./crawler.py", line 82, in downloadPage
    pageText = getPageText(link)
  File "./crawler.py", line 13, in getPageText
    pageText=requests.get(link, cookies={'over18':'1'}).text
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/sessions.py", line 518, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/sessions.py", line 639, in send
    r = adapter.send(request, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/adapters.py", line 488, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
